backbone/src/main/java/org/researchstack/backbone/ui/step/body/FormBody.java

```diff
@@ -50,18 +50,26 @@ public View getBodyView(int viewType, LayoutInflater inflater, ViewGroup parent)
 
         // Iterate through all steps and generate each compact view. Store each StepBody child in a
         // list to iterate over (e.g. within getStepResult())
-        for (QuestionStep questionStep : questionSteps) {
-            StepBody stepBody = createStepBody(questionStep);
-            if(!questionStep.isHidden()) {
+        for (int i = 0; i < questionSteps.size(); i++) {
+            StepBody stepBody = createStepBody(questionSteps.get(i));
+            if(!questionSteps.get(i).isHidden()) {
                 View bodyView = stepBody.getBodyView(VIEW_TYPE_COMPACT, inflater, body);
                 body.addView(bodyView);
+
+                if (i < questionSteps.size() - 1) {
+                    body.addView(getDividerView(inflater, body));
+                }
             }
             formStepChildren.add(stepBody);
         }
 
         return body;
     }
 
+    private View getDividerView(LayoutInflater inflater, ViewGroup parent) {
+        return inflater.inflate(R.layout.rsb_form_step_divider, parent, false);
```

Comment:
I'd inflate this divider once and use the same instance every time. You can either inflate this beforehand or simply do "lazy" like:

```
View getDividerView(...) {
if (this.divider == null) { 
   this.divider = inflater...
}

return this.divider
}
```

Reply:
Can't add the same view object to the linear layout body it cause 
java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.

Reply:
Yeah, you're right. Nevermind, this shouldn't be an issue anyway, we can't have that many views in a form by design so...